### PR TITLE
COMMERCE-291 and COMMERCE-1087 (from Luca)

### PR DESCRIPTION
**Pull request by ethanbustad**
@luca-pellizzon 
From https://github.com/liferay/com-liferay-commerce/pull/619.

### CODE REVIEW

**Comment by ethanbustad** [comment]
ci:test

File: commerce-checkout-web/src/main/java/com/liferay/commerce/checkout/web/internal/util/CommerceCheckoutStepHelper.java [hunk 1]
```diff
@@ -61,12 +67,47 @@ public boolean isActivePaymentMethodCommerceCheckoutStep(
...
+
+		if (commercePaymentMethods.isEmpty()) {
+			return false;
+		}
+
+		if (commercePaymentMethods.size() == 1) {
+			CommercePaymentMethod commercePaymentMethod =
+				commercePaymentMethods.get(0);
+
+			commerceOrder.setCommercePaymentMethodKey(
```
Comment (marco-leo) [inline commerce-checkout-web/src/main/java/com/liferay/commerce/checkout/web/internal/util/CommerceCheckoutStepHelper.java:98]: @luca-pellizzon  can we create a updateCommerceOrderPaymemntMethod in the service?

File: commerce-checkout-web/src/main/java/com/liferay/commerce/checkout/web/internal/util/CommerceCheckoutStepHelper.java [hunk 2]
```diff
@@ -61,12 +67,47 @@ public boolean isActivePaymentMethodCommerceCheckoutStep(
...
+				commercePaymentMethod.getKey());
+
+			commerceOrder = _commerceOrderLocalService.updateCommerceOrder(
```
Comment (marco-leo) [inline commerce-checkout-web/src/main/java/com/liferay/commerce/checkout/web/internal/util/CommerceCheckoutStepHelper.java:101]: @luca-pellizzon always use remote service. Please send a fix in a new pull

**Comment by ethanbustad** [comment]
ci:test

**Pull request merged**